### DISCUSSION TITLE
How to separate between upper div and lower div for overflow-x-scroll

### DISCUSSION THREAD

**Discussion by stackerito** [tailwindlabs/tailwindcss#5122] [Help]
I have the following side bar with a top panel div with the logo and `x` icon: https://play.tailwindcss.com/SPEEjiEKFH

Below that there are 5 divs that can be scrolled from left to right.
But that also scrolls the top div with the panel and I want this panel to stay in place, and only the lower div container (with the 5 inner divs) to scroll

I tried to add `overflow-x-hidden` to the wrapping `div` and adding `overflow-x-scroll` to the `<aside>` element (just like I did with `overflow-y-hidden`) but that didn't work

**Comment by simonswiss** [comment]
Hey!

What you described _should_ work. You could add an `overflow-auto` to the container you want to scroll (the `aside` on your Tailwind Play).

Adding this class will make that element only scroll horizontally.

Here's a play where I have just added that class (+ formatted your makeup with `cmd + S`):

https://play.tailwindcss.com/ngTe1X3GZo

https://user-images.githubusercontent.com/485747/127802752-d2f41a55-4650-44b8-86f8-211170e536f3.mov
